pkg/cmd/cmd.go

```diff
@@ -12,16 +12,19 @@ import (
 
 // Execute runs the main command of the CLI tool.
 func Execute(settings *config.Settings, version string) error {
-	mainCmd := GenerateCmd(settings, version)
+	mainCmd, err := GenerateCmd(settings, version)
+	if err != nil {
+		return err
+	}
 	ctx := context.Background()
 
-	err := mainCmd.ExecuteContext(ctx)
+	err = mainCmd.ExecuteContext(ctx)
```

Comment:
does this still work without assigning the variable? might just be me not understanding go 😋

Reply:
Good question. The return signature of GenerateCMD on line 15 has been changed to include a possible error, so _err_ is assigned there instead. Subsequent functions that return an error do not assign.